src/components/RightSidePanelOptimalRouteInfo/RightSidePanelOptimalRouteIinfo.jsx

```diff
@@ -37,6 +37,8 @@ export const RightSidePanelOptimalRouteIinfo = ({
   const onSelectOptimalRoute = async (optimalRouteId) => {
     try {
       await selectOptimalRoute(optimalRouteId)
+      setSnackbarText('Ruta enviada.')
+      setSnackBarSeverity('success')
```

Comment:
no van estas dos lineas?
pq si BE tira error, y pruebo de nuevo y anda bien, no quedaría con el severity y text del error sino?

Reply:
Yes, tenes razon